lib/iris/coord_categorisation.py

```diff
@@ -32,7 +32,10 @@
 import six
 
 import calendar
-import collections
+try:  # Python 3
+    from collections.abc import Counter
```

Comment:
`Counter` isn't an abc and is not in the `collections.abc` module, this import will always fail. I don't think this module needs any updates.

Reply:
Oops thanks for the pick up!